openquake/baselib/__init__.py

```diff
@@ -47,8 +47,22 @@ def __getattr__(self, key):
 config = DotDict()  # global configuration
 
 
-def _read(*paths, **validators):
-    # load the configuration file by looking at the given paths
+def read(*paths, **validators):
+    """
+    Load the configuration, make each section available in a separate dict.
+
+    The configuration location can specified via an environment variable:
+       - OQ_CONFIG_FILE
```

Comment:
`s/OQ_CONFIG_FILE/OQ_CONFIG_FILE_VAR/g`

or if you like more `OQ_CONFIG_FILE` (I do) we can change it in the code

Reply:
Changed in the code to be consistent with the documented name.